scripts/qa_crowbarsetup.sh

```diff
@@ -2829,15 +2866,15 @@ function set_proposalvars
 function set_noproxyvar
 {
     [[ $http_proxy ]] || [[ $https_proxy ]] || return 0
-    [[ $no_proxy =~ "localhost" ]] || no_proxy="127.0.0.1,localhost,$no_proxy"
+    [[ $no_proxy =~ "localhost" ]] || no_proxy="127.0.0.1,localhost,::1,$no_proxy"
     [[ ! $adminip ]] || [[ $no_proxy =~ "$adminip" ]] || no_proxy="$adminip,$no_proxy"
     [[ ! $cloudfqdn ]] || [[ $no_proxy =~ "$cloudfqdn" ]] || no_proxy="$cloudfqdn,$no_proxy"
     export no_proxy="${no_proxy%,}";
     if [[ ! $net_public ]] || [[ $no_proxy =~ $net_public ]] ; then
         return 0 # only apply this once
     fi
     local ips
-    printf -v ips '%s,' $net_public.{1..254}
+    printf -v ips '%s,' $net_public${ip_sep}{1..254}
```

Comment:
I wonder why this line was added in the first place, and I've been wondering what was dumping an awful lot of IPs to stdout during mkcloud runs

Reply:
And in fact, this is totally wrong for IPv6. This will be *WAY* too wide with $net_public:{1..254}, and we probably want to do something else when IPv6 is set, since we want our range to be 0000 to ffff and we need to use :: (think about it)

Reply:
Yeah no idea,, we could have at least put it behind a debug if that's what it was for. But figure that is out of scope of this change.

Reply:
Yeah that's true. In the commit message I mention I've left the last octet the same between ipv4 and ipv6 to keep the last octets similar between ipv4 and ipv6 runs, to make it easier for people to debug. (though this didn't work for the admin node, thanks SLAAC). (see also the network limitations in the ipv6 network.json to this effect). But yes.. there will be much more printing of IPs if we use 0000-ffff